src/components/NewsletterSignup/NewsletterSignup.stories.tsx

```diff
@@ -272,3 +284,15 @@ export const ComponentStates: Story = {
     </VStack>
   ),
 };
+
+/* To fix focus issue where the page focuses on the last NewsletterSignup 
```

Comment:
Yes, this is a bit hacky but it works. Let's keep it for now knowing it could break later. Can you add to the comment that it's expected behavior for the different screen views to have focus, so this is to better view the docs when multiple newslettersignup components are on a single page but this should not happen in a real app.

Reply:
I added some more comment. It is def hacky 😎